modelforge/tests/test_utils.py

```diff
@@ -73,10 +79,12 @@ def test_gaussian_rbf(RBF):
     assert gaussian_rbf.n_rbf == n_rbf
 
     # Test that the cutoff distance is correct
-    assert gaussian_rbf.cutoff == cutoff
+    assert gaussian_rbf.cutoff == cutoff.value_in_unit_system(unit.md_unit_system)
```

Comment:
I think we should use openff units, as we've bee using those throughout the code already. 

we would use the syntax:

`assert gaussian_rbf.cutoff == cutoff.is_compatible_with(unit.nanometer)`